README.md

```diff
@@ -121,10 +121,6 @@ async def print_my_user(token):
 asyncio.run(print_my_user("user token here"))
```

Comment:
Should we update this codeblock to use the new `ClientCredentialsStrategy`?

Reply:
Don't think it'd quite fit there

Reply:
Why not?

Reply:
since it's for an awfully specific oauth2 flow